bubble_sort.rb

```diff
@@ -0,0 +1,31 @@
+# First bubble-sort method for integers
+def bubble_sort(arr)
+  i = 0
+  length = arr.length
+  while i < length
+    arr[i], arr[i + 1] = arr[i + 1], arr[i] if arr[i] > arr[i + 1]
+    i += 1
+    if i == length - 1
+      i = 0
+      length -= 1
+    end
+  end
+  arr
+end
+puts bubble_sort([4, 3, 5, 1, 2])
```

Comment:
- [ ] About good practices, please do not commit debugging code (generally include `puts`, `print` or `p`). Later in the course, you will learn how to add automated tests where you will be able to put your examples ;) Please delete this line
![image](https://user-images.githubusercontent.com/53256139/99864765-cfeee600-2b6a-11eb-8ec7-a84406a5e894.png)